Cargo.toml

```diff
@@ -16,13 +16,13 @@ num-traits = "0.2.8"
 thiserror = "1.0.4"
 
 [build-dependencies]
-cbindgen = "0.22.0"
+cbindgen = { version = "0.22.0", default-features=false, optional = true}
```

Comment:
note also the `default-features=false`, which avoids building `clap` in this case. 

I'm trying to get better about disabling unnecessary features, but it's hard to remember.